src/fidesctl/core/config/database_settings.py

```diff
@@ -0,0 +1,72 @@
+import os
+from typing import Dict, Optional
+
+from fideslib.core.config import DatabaseSettings
+from pydantic import Field, PostgresDsn, validator
+
+
+class FidesctlDatabaseSettings(DatabaseSettings):
+    """Configuration settings for Postgres."""
+
+    @staticmethod
+    def default() -> "FidesctlDatabaseSettings":
+        return FidesctlDatabaseSettings(
+            user="postgres",
+            password="fidesctl",
+            server="fidesctl-db",
+            port="5432",
+            db="fidesctl",
+            test_db="fidesctl_test",
+        )
```

Comment:
fidesctl has logic which assumes that all required configs have defaults set. This doesn't seem to be something fideslib expects so we have to add defaults like this.

Reply:
using the default python constructor wasnt working well with pydantic